### ISSUE TITLE
3.4.1.3.2 Page List in EPUB A11Y 1.1

### ISSUE THREAD

**Issue by t-kudou** [mattgarrish/epub-revision#2220]
https://www.w3.org/TR/epub-a11y-11/#sec-page-list

The first line of Meeting this Objective, "MUST have a page list", but the next line "SHOULD include links to all pages", which is strange (contradictory?).
The following is a list of the most common problems. If there is no page list, it reads that an empty page list should be provided.
In the existing EPUB, if there is no page list, the page list itself does not exist in the Navigation Document.
This part of the description should be changed due to compatibility issues.

**Comment by mattgarrish** [comment]
> The first line of Meeting this Objective, "MUST have a page list", but the next line "SHOULD include links to all pages"

Just to be clear, the requirement is contingent on what is reproduced:

> EPUB Creators SHOULD include links to all pages of content reproduced from the source (i.e., they do not have to provide links for blank pages or content not reproduced in the digital edition).

This could be stated as a MUST, though, without any extra burden on publishers. The key here is "reproduced", so there is a built-in exception for content not represented in the digital version.

I can't remember if there were other cases where a book might not represent all its pages in the page list, though.

Thoughts @avneeshsingh @GeorgeKerscher?

**Comment by clapierre** [comment]
Would a choose your own adventure apply here you might not include all pages in this case, maybe the main storyline only.  But I agree the main use case was blank pages could be skipped from the page list.

**Comment by mattgarrish** [comment]
> Would a choose your own adventure apply here you might not include all pages in this case

I think you'd still want the complete page list. Even if you're not reading sequentially, you may want to go back to a specific page you knew you were on.

The problem with these books usually revolves around the spine and having to at least list the content documents as non-linear. Some people want to force users to follow links by excluding the documents, but that's a rabbit hole we don't need to go down again.

**Comment by clapierre** [comment]
By making this a "MUST" like the pull request has added also in my opinion would satisfy

2.4.5 Multiple Ways Level AA

There must be at least two ways that allow a user can navigate through the content. EPUB publications minimally meets this requirement if all content documents are listed in the spine and there is a navigation document with links to the major sections.

The addition of a page list or index can also be counted in meeting this criterion.
